src/main/resources/assets/twilightforest/patchouli_books/guide/en_us/entries/treasures/minotaur_axes.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Minotaur Axes",
+  "icon": "twilightforest:minotaur_axe",
+  "category": "treasures",
+  "advancement": "twilightforest:alt/treasures/minotaur_axes",
+  "sortnum": 5030,
+  "pages": [
+    {
+      "type": "text",
+      "text": "Finely forged weapons wielded by Minotaurs and the Minoshroom. At full force, they can be devastating."
```

Comment:
IIRC the stats on these axes are same as Vanilla axe counterparts. Only difference being able to charge with them.

Perhaps write about how they seem like ordinary axes but feel lightweight to go charging full-force